Doc/library/logging.rst

```diff
@@ -1131,7 +1131,7 @@ functions.
    if no handlers are defined for the root logger.
 
    This function does nothing if the root logger already has handlers
-   configured for it.
+   configured except when a keyword argument *force* is set as true.
```

Comment:
... has handlers configured, unless the keyword argument *force* is set to ``True``.